### PR TITLE
Implement oracle script table UI (both revisions and requests) 

### LINKED ISSUE #853: Implement oracle script table UI (both revisions and requests)

### PR DESCRIPTION
**Pull request by Benzbeeb**
Fixed #853

### CODE REVIEW

File: scan/src/components/oracle-script/OracleScriptRequestTable.re [hunk 1]
```diff
@@ -136,6 +150,13 @@ let make = () => {
               })
             ->Array.of_list
             ->React.array}
+           <VSpacing size=Spacing.lg />
+           <VSpacing size=Spacing.lg />
+           <Pagination
+             currentPage=page
+             pageCount
+             onPageChange={newPage => setPage(_ => newPage)}
+           />
```
Comment (evilpeach) [inline scan/src/components/oracle-script/OracleScriptRequestTable.re]: No need to add pagination for now.

File: scan/src/components/oracle-script/OracleScriptRequestTable.re [hunk 2]
```diff
@@ -25,6 +25,8 @@ type request_t = {
 
 [@react.component]
 let make = () => {
+  let (page, setPage) = React.useState(_ => 1);
+  let pageCount = 10;
```
Comment (evilpeach) [inline scan/src/components/oracle-script/OracleScriptRequestTable.re]: remove

File: scan/src/components/OracleScriptRevisionTable.re [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+      flexDirection(`column),
+      alignItems(`center),
+    ]);
+};
+
+[@react.component]
+let make = (~revisions: list(OracleScriptHook.OracleScript.revision_t)) => {
+  let (page, setPage) = React.useState(_ => 1);
+  let pageCount = 10;
+
```
Comment (evilpeach) [inline scan/src/components/OracleScriptRevisionTable.re]: remove as well

File: scan/src/components/OracleScriptRevisionTable.re [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+              })
+            ->Array.of_list
+            ->React.array}
+           <VSpacing size=Spacing.lg />
+           <VSpacing size=Spacing.lg />
+           <Pagination
+             currentPage=page
+             pageCount
+             onPageChange={newPage => setPage(_ => newPage)}
+           />
```
Comment (evilpeach) [inline scan/src/components/OracleScriptRevisionTable.re]: remove

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by evilpeach — APPROVED** [review]

**Review by prin-r — APPROVED** [review]

**Pull request merged**